src/test/resources/database/add-books-categories-dependencies.sql

```diff
@@ -0,0 +1,5 @@
+INSERT INTO books_categories (book_id, category_id)
+values (1, 1);
```

Comment:
```suggestion
VALUES (1, 1);
```
use upper case for  all SQL operators (fix in other places)